scripts/tests.sh

```diff
@@ -109,9 +109,18 @@ function run_eth()
     sleep 2
 }
 
+function check_eth() {
+    printTask "Running IPC tests with $ETH_PATH..."
+    if ! hash $ETH_PATH 2>/dev/null; then
```

Comment:
Why not use `if [ ! -x "$ETH_PATH" ]; ...`? hash seems to be a little bit too intrusive.

Reply:
Ah sorry, this is also for the non-CI case.

Reply:
I like the idea of using a simpler check. However, the code you suggest -as-is- doesn't work. This does however:
```
if ! [ -x "$(command -v $ETH_PATH)" ]; then
```
Let me know if that will work for you and I'll update the PR.

Reply:
It's fine - I already merged it as is :) Thanks a lot!